Makefile

```diff
@@ -242,9 +242,8 @@ localnet_regenesis: check_yq acc_initialize_pubkeys_warn_message ## Regenerate t
 	@echo "Initializing chain..."
 	@set -e ;\
 	ignite chain init --skip-proto ;\
-	mkdir -p $(POKTROLLD_HOME)/config/ ;\
```

Comment:
Why not keep `mkdir -p` but remove ` /config/` from the end?

Reply:
`cp -r ${HOME}/.poktroll/config $(POKTROLLD_HOME)/` creates the config directory. Having them both is redundant.

We can leave `mkidr -p` and have `cp -r ${HOME}/.poktroll/config/* $(POKTROLLD_HOME)/config/`

They achieve the same, except the later may skip dot prefixed files.